src/ControlSystem/Protocols/ControlError.hpp

```diff
@@ -24,6 +24,11 @@ namespace control_system::protocols {
 ///   the example shown here:
 /// - a `static constexpr size_t expected_number_of_excisions` which specifies
 ///   the number of excisions necessary in order to compute the control error.
+/// - a type alias `object_centers` to a `domain::object_list` of
```

Comment:
How about you just require a type alias `const_global_cache_tags` and collect those in the parallel component or in the init action? Then there's less magic involved, and this behaves like other places in the code where we collect global cache tags.

Reply:
The control errors need to be agnostic to how the center tags were put into the cache. The control errors grab the base tag. My plan for this was in the initialization action in the future choose what actual simple tags (that are derived from the base tag) to use depending on measurements.